docs/13-walkthroughs/6-publish-and-deploy.md

```diff
@@ -145,75 +70,99 @@ To modify, edit `zarf.yaml` and re-run `zarf package create .`
 
-```bash
-# go home so we don't clobber our currently local built package
-$ cd ~
-$ mkdir -p zarf-packages && cd zarf-packages
+<iframe src="/docs/walkthroughs/publish_and_deploy_pull.html" width="100%" height="450px"></iframe>
+
+
+## Removal
+
+1. Use the `zarf package list` command to get a list of the installed packages.  This will give you the name of the games package to remove it.
+
+<iframe src="/docs/walkthroughs/publish_and_deploy_list.html" width="100%"></iframe>
+
+2. Use the `zarf package remove` command to remove the `helm-oci-chart` package.  Don't forget the `--confirm` flag.  Otherwise you'll receive an error.
+
+<iframe src="/docs/walkthroughs/publish_and_deploy_remove.html" width="100%" height="115px"></iframe>
+
+## Troubleshooting
+
+### Failed to publish package: version is required for publishing
+
+#### Example
+
+<iframe src="/docs/walkthroughs/troubleshoot_version_required_publish.html" width="100%" height="130px"></iframe>
+
+#### Remediation
+
+You attepted to publish a package with no version metadata.
+
+<iframe src="/docs/walkthroughs/troubleshoot_version_required_no_version.html" width="100%" height="300px"></iframe>
+
+1. Open the zarf.yaml file.
+2. Add a version attribute to the [package metadata](https://docs.zarf.dev/docs/user-guide/zarf-schema#metadata)
+3. Recreate the package with the `zarf package create` command.
+4. Publish the package.  The filename will now have the version as part of it.
+
+### Failed to publish, http: server gave HTTP response to HTTPS client
+
+#### Example
+
+<iframe src="/docs/walkthroughs/troubleshoot_insecure_registry.html" width="100%" height="425px"></iframe>
+
+#### Remediation
+
+You attempted to publish a package to an insecure registry, using http instead of https.
+
+1. Use the `--insecure` flag.  This is not suitable for production workloads.
+
+### Unable to connect to the Kubernetes cluster.
+
+#### Example
+
+<iframe src="/docs/walkthroughs/troubleshoot_unreachable.html" width="100%" height="200px"></iframe>
+
+#### Remediation
+
+If you receive this error, either you don't have a Kubernetes cluster, your cluster is down, or your cluster is unreachable.
+
+1. Check your kubectl configuration, then try again.  For more information about kubectl configuration see [Configure Access to Multiple Clusters](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) from the Kubernetes documentation.
+
+If you need to setup a cluster, you can perform the following.
+
+1. Deploy a Kubernetes cluster with the [Creating a K8s Cluster with Zarf](./4-creating-a-k8s-cluster-with-zarf.md) walkthrough.
+2. Perform the [Initialize a cluster](./1-initializing-a-k8s-cluster.md) walkthrough.
+
+After that you can try deploying the package again.
+
+### Secrets "zarf-state" not found.
+
+#### Example
+
+<iframe src="/docs/walkthroughs/troubleshoot_uninitialized.html" width="100%" height="250px"></iframe>
```

Comment:
Same with remove - there are common troubleshooting steps that should get their own page.

Reply:
Agreed.  Will  add it to my notes, if there's not an issue for a common troubleshooting page, let's make one.